features/cart/shopping_cart/changing_dispalay_currency_of_cart.feature

```diff
@@ -1,12 +1,13 @@
 @shopping_cart
-Feature: All of my cart's values get updated to the currency of my choosing
-    In order to know how much I have to pay in my currency
+Feature: Changing display currency of the cart
+    In order to know estimated price for foreign currencies
     As a Visitor
-    I want to see every cash amount in my chosen currency
+    I want to see every cash amount rounded to my chosen currency
 
     Background:
         Given the store operates on a single channel in "United States"
-        And that channel allows to shop using the "GBP" currency with exchange rate 2.00
+        And that channel allows to shop using the "GBP" currency
+        When the store has an exchange rate 2.0 with source currency "US Dollar" and target currency "British Pound"
```

Comment:
When the exchange rate of "US Dollar" to "British Pound" is 2.0

Or anything that sounds like human language and not "for-transformers" dialect. :/